quickstarts/istio/config.yml

```diff
@@ -1,10 +1,13 @@
 id: 36aa41e3-2d2a-41d7-844a-083f00c5d65e
-name: istio
+name: istio (Deprecated)
```

Comment:
```suggestion
name: istio
```

Reply:
Just removing this as I think it will look a bit odd in the UI, and the deprecation notice is in the summary.